onadata/libs/utils/logger_tools.py

```diff
@@ -624,6 +626,8 @@ def save_submission(
     if getattr(instance, 'defer_counting', False):
         # Remove the Python-only attribute
         del instance.defer_counting
+        update_xform_daily_counter(instance=instance, created=True)
+        update_xform_monthly_counter(instance=instance, created=True)
         update_xform_submission_count(instance=instance, created=True)
         update_user_submissions_counter(instance=instance, created=True)
```

Comment:
this and its associated `SubmissionCounter` model could be removed per https://hackmd.io/_c5ylWuEQ6anJcMAj0odyQ#Questions